.github/workflows/build_nilrt.yml

```diff
@@ -134,8 +134,8 @@ jobs:
           message(FATAL_ERROR "Build failed")
         endif()
 
-    - name: Tar Build Files
-      if: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/releases') }}
+    - name: Tar Development Build Files
+      if: ${{ github.ref == 'refs/heads/main' }}
       run: >-
         tar -cvf NILinuxRealTime-x86_64.tar
```

Comment:
If we make these -cvzf will Github still automatically zip them?

Reply:
Yes. All artifact downloads are zipped, even if you've already zipped them once. Just an annoying thing about how GitHub hosts them.

Reply:
Probably still worth using -cvzf and ending files with .tar.gz I guess. We'll still have to unzip it to get down to the .tar.gz, but that's not too bad

Reply:
I've gone back and forth on this a couple of times. I don't really like having the extra layer of compression (Linux can handle zip files without issue I believe) but after thinking about it more, the extra compression to save space on our artifact storage is probably worthwhile even if we currently have unlimited space for the public repo.

Fixed.

Reply:
The more important aspect of this is the ease of creating the release. I don't want to have to unzip the files then tar.gz them up. That adds an extra source of possible user error. If we zip up the compressed .tar.gz, then unzipping will give you the right thing to upload to the release.